cf_units/tests/unit/test__udunits2.py

```diff
@@ -26,7 +27,10 @@ class Test_get_system(unittest.TestCase):
     """
 
     def test_read_xml(self):
-        system = _ud.read_xml()
+        try:
+            system = _ud.read_xml()
+        except _ud.UdunitsError:
+            system = _ud.read_xml(get_xml_path())
```

Comment:
This pattern requires to be adopted in order to successfully test a `wheel` installation.